models/repo/license.go

```diff
@@ -27,7 +27,10 @@ type RepoLicense struct { //revive:disable-line:exported
 type RepoLicenseList []*RepoLicense //revive:disable-line:exported
 
 func (rll RepoLicenseList) StringList() []string {
-	var licenses []string
+	if len(rll) == 0 {
+		return nil
+	}
+	licenses := make([]string, 0, len(rll))
```

Comment:
```suggestion
	var licenses []string
```

Let's keep it simple as before 🙏